packages/app/src/providers/hsds/hsds-api.ts

```diff
@@ -184,8 +205,20 @@ export class HsdsApi extends ProviderApi {
     return data.value;
   }
 
+  private async fetchAttributeWithValue(
+    entityCollection: HsdsCollection,
```

Comment:
The endpoint needs to know the HSDS entity's `collection`, which is like the `type` but with an `s`... So far we were discarding this information which comes from the link resolution phase. Instead of trying to guess the collection from the type, which is tricky for unresolved entities... I decided to keep the `collection` around like we do for the `id`. That's why you see it a bit everywhere in the diff.